Documentation/contributing/development/images.rst

```diff
@@ -20,22 +20,17 @@ contains your local changes.
 
 ::
 
-    DOCKER_DEV_ACCOUNT=quay.io/myaccount DOCKER_IMAGE_TAG=jane-developer-my-fix make dev-docker-image
+    ARCH=amd64 DOCKER_FLAGS=--push DOCKER_DEV_ACCOUNT=quay.io/myaccount DOCKER_IMAGE_TAG=jane-developer-my-fix make dev-docker-image
```

Comment:
When `ARCH` is defined, specifying `DOCKER_FLAGS=--push` is not necessary - https://github.com/cilium/cilium/blob/master/Makefile.docker#L28. Although it's not clear to me why the two options are not decoupled.

Reply:
Ah, thanks, I missed that. It first complained `--push` was missing so I added that, then it complained `--push` wasn't compatible with `--load`, so I added `ARCH` to remove `--load` :-/